apps/vscode/src/package.json

```diff
@@ -587,7 +647,15 @@
           "id": "nxProjects",
           "name": "Projects",
           "when": "isNxWorkspace || isAngularWorkspace"
-        }
```

Comment:
meh, got a ["welcome view"](https://code.visualstudio.com/api/references/contribution-points#contributes.viewsWelcome) showing up along with the nearly empty 'Select workspace' treeview when an angular or Nx project isn't open. Might work for "what is this?"
![Screen Shot 2021-05-11 at 2 49 06 PM](https://user-images.githubusercontent.com/2250413/117875899-535ed980-b268-11eb-9211-c14f29ab6c7f.png)

Reply:
Can we move this up above the generate and run target view?

Reply:
Yeah, looks like we can! Great idea